solutions/cses-2428.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: cses-2428
+source: CSES
+title: Subarray Distinct Values
+author: Sofia Yang
+---
+
+## Explanation
+
+Use a sliding window and keep a map to record the frequency of each element in the array.
```

Comment:
the use of map here is equivalent to mulitset.

Reply:
Ok, I changed it.